pkg/v1/resource/certconfig/create_test.go

```diff
@@ -196,12 +196,12 @@ func Test_newCreateChange(t *testing.T) {
 			},
 			currentState: []*v1alpha1.CertConfig{
 				newCertConfig("cluster-1", certs.APICert),
-				newCertConfig("cluster-1", certs.CalicoCert),
+				newCertConfig("cluster-1", certs.CalicoEtcdClientCert),
```

Comment:
Things are in so broken state that we need that cert. Please update `aws-sdk-go` with `dep ensure -update github.com/aws/aws-sdk-go`.

Reply:
dep ensure update was run but there’s a constraint to v1.14.7 because of e2eclients.

I do not see the relation between calico certs and aws sdk.

Reply:
We just can't vendor newest `certs@master`. Maybe I should pin it.

Reply:
I tried to pin `certs@e91f12dea59971a68d08a48479014e33d1b90540`, which is the version used on master. But it leads to other constraints conflicts:

```
$ dep ensure
Solving failure: No versions of github.com/giantswarm/guestcluster met constraints:
        master: Could not introduce github.com/giantswarm/guestcluster@master, as it has a dependency on github.com/giantswarm/certs with constraint master, which has no overlap with existing constraint e91f12dea59971a68d08a48479014e33d1b90540 from (root)
        master: Could not introduce github.com/giantswarm/guestcluster@master, as it has a dependency on github.com/giantswarm/certs with constraint master, which has no overlap with existing constraint e91f12dea59971a68d08a48479014e33d1b90540 from (root)
        teemow-update-apiextensions: Could not introduce github.com/giantswarm/guestcluster@teemow-update-apiextensions, as it is not allowed by constraint master from project github.com/giantswarm/cluster-operator.
```

Reply:
When you want to do it like that it could work with `override` instead of `constraint`.

Reply:
nice thanks, didn't knew about this

Reply:
Thanks. I reverted those cert changes.

Reply:
Updating `certs` repo here is on my list so I no need for creating any TODO.